docs/usage.md

```diff
@@ -6,14 +6,18 @@
 
 ## Introduction
 
-<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
+General usage:
+
+```bash
+nextflow run nf-core/pgdb -profile <docker/singularity/podman/conda/institute> --ensembl_name homo_sapiens
```

Comment:
I think it's a bit confusing to have this as general usage, with a totally different command just underneath labelled as _"the typical command"_.

Maybe worth rewriting this part of the docs a little to describe what the workflow does / how to use it in more general terms? Or at least just pick one? Basically rather than filling in the template it would be good if this was viewed as a stand-alone documentation prose from the viewpoint of a newcomer.